src/en/aurora/src/eu/kanade/tachiyomi/extension/en/aurora/Aurora.kt

```diff
@@ -0,0 +1,228 @@
+package eu.kanade.tachiyomi.extension.en.aurora
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.Jsoup
+import rx.Observable
+import java.lang.Exception
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+/**
+ *  @author THE_ORONCO <the_oronco@posteo.net>
+ */
+
+class Aurora : HttpSource() {
+
+    override val name = "Aurora"
+    private val authorName = "OSP-Red"
+    override val baseUrl = "https://comicaurora.com"
+    private val chapterListUrl = "$baseUrl/archive"
+    override val lang = "en"
+    override val supportsLatest = false
+
+    override fun chapterListRequest(manga: SManga): Request {
+        return GET(chapterListUrl)
+    }
+
+    /**
+     *
+     */
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val doc = response.asJsoup()
+
+        val chapters: List<SChapter> =
+            doc.select("li.blocks-gallery-item")
+                // filter out all elements that have a chapter cover and a link to the actual chapter
+                .filter { element ->
+                    element.select("a[href]").size > 0 &&
+                        element.select("img[src]").size > 0
+                }
```

Comment:
The filter seems unnecessary.
```suggestion
            doc.select(".blocks-gallery-item")
```